framework/mongo/validators.py

```diff
@@ -7,3 +7,31 @@
 def string_required(value):
     if value is None or value == '':
         raise ValidationValueError('Value must not be empty.')
+    return True
+
+
+def choice_in(choices, ignore_case=False):
```

Comment:
It's a similar behavior to how regexes operate: it looks for an *exact* match unless you specifically tell it otherwise.